source/Calamari/Commands/DownloadPackageCommand.cs

```diff
@@ -133,16 +134,6 @@ public override int Execute(string[] commandLineArguments)
             return 0;
         }
 
-        static ICredentials GetFeedCredentials(string feedUsername, string feedPassword)
```

Comment:
The requirements for authentication are different for each downloader implementation (e.g. GitHub doesn't require the username) so the checking and marshalling of the provided values into a valid credential is delegated to each implementation.